2025-04-21 06:03:37,314 INFO    MainThread:28540 [wandb_setup.py:_flush():67] Current SDK version is 0.19.9
2025-04-21 06:03:37,315 INFO    MainThread:28540 [wandb_setup.py:_flush():67] Configure stats pid to 28540
2025-04-21 06:03:37,315 INFO    MainThread:28540 [wandb_setup.py:_flush():67] Loading settings from /Users/kovarthanan/.config/wandb/settings
2025-04-21 06:03:37,315 INFO    MainThread:28540 [wandb_setup.py:_flush():67] Loading settings from /Users/kovarthanan/Acedamic/City, University of London/INM705 - Image Analysis/AI-Generated-Image-Detector/wandb/settings
2025-04-21 06:03:37,315 INFO    MainThread:28540 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-04-21 06:03:37,315 INFO    MainThread:28540 [wandb_init.py:setup_run_log_directory():662] Logging user logs to /Users/kovarthanan/Acedamic/City, University of London/INM705 - Image Analysis/AI-Generated-Image-Detector/wandb/run-20250421_060337-ckl9rn0z/logs/debug.log
2025-04-21 06:03:37,315 INFO    MainThread:28540 [wandb_init.py:setup_run_log_directory():663] Logging internal logs to /Users/kovarthanan/Acedamic/City, University of London/INM705 - Image Analysis/AI-Generated-Image-Detector/wandb/run-20250421_060337-ckl9rn0z/logs/debug-internal.log
2025-04-21 06:03:37,315 INFO    MainThread:28540 [wandb_init.py:monkeypatch_ipython():581] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x103fbc8b0>
2025-04-21 06:03:37,315 INFO    MainThread:28540 [wandb_init.py:init():781] calling init triggers
2025-04-21 06:03:37,315 INFO    MainThread:28540 [wandb_init.py:init():786] wandb.init called with sweep_config: {}
config: {'epochs': 5, 'batch_size': 32, 'lr': 5e-05, '_wandb': {}}
2025-04-21 06:03:37,315 INFO    MainThread:28540 [wandb_init.py:init():809] starting backend
2025-04-21 06:03:37,316 INFO    MainThread:28540 [wandb_init.py:init():813] sending inform_init request
2025-04-21 06:03:37,351 INFO    MainThread:28540 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2025-04-21 06:03:37,352 INFO    MainThread:28540 [wandb_init.py:init():823] backend started and connected
2025-04-21 06:03:37,358 INFO    MainThread:28540 [wandb_run.py:_label_probe_notebook():1267] probe notebook
2025-04-21 06:03:37,362 INFO    MainThread:28540 [wandb_init.py:init():915] updated telemetry
2025-04-21 06:03:37,384 INFO    MainThread:28540 [wandb_init.py:init():939] communicating run to backend with 90.0 second timeout
2025-04-21 06:03:38,365 INFO    MainThread:28540 [wandb_init.py:init():1014] starting run threads in backend
2025-04-21 06:03:38,442 INFO    MainThread:28540 [wandb_run.py:_console_start():2454] atexit reg
2025-04-21 06:03:38,442 INFO    MainThread:28540 [wandb_run.py:_redirect():2306] redirect: wrap_raw
2025-04-21 06:03:38,442 INFO    MainThread:28540 [wandb_run.py:_redirect():2371] Wrapping output streams.
2025-04-21 06:03:38,443 INFO    MainThread:28540 [wandb_run.py:_redirect():2394] Redirects installed.
2025-04-21 06:03:38,445 INFO    MainThread:28540 [wandb_init.py:init():1056] run started, returning control to user process
2025-04-21 06:03:38,446 INFO    MainThread:28540 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-04-21 06:03:38,446 INFO    MainThread:28540 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-04-21 06:03:38,452 INFO    MainThread:28540 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-04-21 06:03:38,453 INFO    MainThread:28540 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-04-21 06:03:38,453 INFO    MainThread:28540 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-04-21 06:03:38,458 INFO    MainThread:28540 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-04-21 06:03:39,813 INFO    MainThread:28540 [wandb_watch.py:_watch():71] Watching
2025-04-21 06:03:39,814 INFO    MainThread:28540 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-04-21 06:03:39,814 INFO    MainThread:28540 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-04-21 06:03:39,818 INFO    MainThread:28540 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-04-21 06:03:39,819 INFO    MainThread:28540 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-04-21 06:03:39,819 INFO    MainThread:28540 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-04-21 06:03:39,827 INFO    MainThread:28540 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-04-21 06:27:47,123 INFO    MainThread:28540 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-04-21 06:27:47,125 INFO    MainThread:28540 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-04-21 06:27:47,166 INFO    MainThread:28540 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-04-21 06:28:16,180 INFO    MainThread:28540 [wandb_run.py:_finish():2189] finishing run avok/INM705-Image/ckl9rn0z
2025-04-21 06:28:16,185 INFO    MainThread:28540 [jupyter.py:save_ipynb():384] not saving jupyter notebook
2025-04-21 06:28:16,185 INFO    MainThread:28540 [wandb_init.py:_jupyter_teardown():563] cleaning up jupyter logic
2025-04-21 06:28:16,185 INFO    MainThread:28540 [wandb_run.py:_atexit_cleanup():2419] got exitcode: 0
2025-04-21 06:28:16,186 INFO    MainThread:28540 [wandb_run.py:_restore():2401] restore
2025-04-21 06:28:16,186 INFO    MainThread:28540 [wandb_run.py:_restore():2407] restore done
2025-04-21 06:30:22,552 INFO    MainThread:28540 [wandb_run.py:_footer_history_summary_info():4064] rendering history
2025-04-21 06:30:22,554 INFO    MainThread:28540 [wandb_run.py:_footer_history_summary_info():4096] rendering summary
2025-04-21 06:30:22,555 INFO    MainThread:28540 [wandb_run.py:_footer_sync_info():4025] logging synced files
